<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BSynergyWidget</class>
 <widget class="QWidget" name="BSynergyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BSynergy</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="m_startGroupBox">
     <property name="title">
      <string>Start</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QRadioButton" name="m_serverRadioButton">
        <property name="text">
         <string>Server</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="m_configureServer">
        <property name="text">
         <string>Konfiguriere Server</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="m_clientRadioButton">
        <property name="text">
         <string>Client</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="m_ipLabel">
        <property name="text">
         <string>IP:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="m_ipLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>BSynergy kann als Server oder als Client gestartet werden. Wenn Sie BSynergy als Server starten, bedeutet das, Sie eine Konfiguration erstellen müssen in welchen Zusammenhang die Computer sich untereinander kennen, wenn sie dieses getan haben, können sich die entsprechenden Clients verbinden. Wenn Sie BSynergy als Client starten, so müssen sie die jeweilige IP des Servers eintragen und sich verbinden.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>37</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="m_startButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_closeButton">
         <property name="text">
          <string>Beenden</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_clientRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_ipLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_closeButton</sender>
   <signal>clicked()</signal>
   <receiver>BSynergyWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>545</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
